__tests__/contracts.js

```diff
@@ -999,3 +1000,48 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
     expect(checked.alreadyDeployed)
         .toBeTruthy();
 });
+test('do_tvm_transfer() should carry all the remaining balance of the current smart contract to another address', async () => {
+    const { contracts, crypto, queries } = tests.client;
+    const giverKeys = await crypto.ed25519Keypair();
+
+    const transferPackage = TransferContractPackage[2];
```

Comment:
Only ABI v2? But you added contract to abi_v1 folder too. Why not using them?

Reply:
fixed